### ISSUE TITLE
Release 4.3.5 - Release Candidate 1 - External integrations modules

### ISSUE THREAD

**Issue by AdriiiPRodri** [wazuh/wazuh#13982]
The following issue aims to perform all the required testing for the current release candidate to ensure the modules (AWS, GCP, Azure and DockerListener) works as expected, report the results, and open new issues for any encountered errors.

## Modules tests information

|||
|----------------------------------|------ |
| **Main release candidate issue** | #13966     |
| **Version**                      | 4.3.5 |
| **Release candidate #**          | RC1   |
| **Tag**                          | [v4.3.5-rc1](https://github.com/wazuh/wazuh/tree/v4.3.5-rc1) |
| **Previous modules tests issue** | -    |

## Test report procedure

All test results must have one the following statuses: 
|                                  |                                                        |
|-----------------|-------------------------------------------------------------------------|
| :green_circle:  | The test passed successfully. 										    |
| :yellow_circle: | The test passed but some enhancements need to be applied to the module. |
| :red_circle:    | The test failed.                                                        |
| :white_circle:  | The test does not apply.                                                |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause. It must be included in the `Fixes` section of the current release candidate main issue.

The resulting logs for the tests must be included in the status report so it can be used by the auditors to dig deeper into any possible failures and details.

## Conclusions

| **Module**      | **Status**  |**Notes**|
|-----------------|-------------|---------|
| AWS             | :white_circle: Skipped | No changes from previous release |
| GCP             | :white_circle: Skipped | No changes from previous release |
| Azure           | :white_circle: Skipped | No changes from previous release |
| Docker Listener | :green_circle: Passed |  |

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.
- [x] @davidjiglesias 
- [x] @CarlosRS9

**Comment by CarlosRS9** [comment]
# DockerListener test results

The tests were carried out using both a Wazuh manager with 3.9 and a Wazuh agent with Python 3.6. both using the following configuration:
```
<wodle name="docker-listener">
    <interval>30s</interval>
    <attempts>5</attempts>
    <run_on_start>yes</run_on_start>
    <disabled>no</disabled>
</wodle>
```

The [logall option](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#logall) must be enabled in the `ossec.conf` file before starting the tests.

### Test cases Python 3.6

#### Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>

</details>


#### Test docker commands

**Run a container with no cache available**
```
docker run ubuntu
```

<details><summary>Archives.json output</summary>


</details>

**Run the same image, this time having the image in cache**

<details><summary>Archives.json output</summary>


</details>

**Remove the image**

<details><summary>Archives.json output</summary>


</details>

**Execute an application in a container**
Run `docker run -td ubuntu bash`

<details><summary>Archives.json output</summary>


</details>

**Kill the container previously launched**
Run `docker kill $(docker ps -q)`.

<details><summary>Archives.json output</summary>


</details>


### Test cases Python 3.9

#### Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>

</details>


#### Test docker commands

**Run a container with no cache available**
```
docker run ubuntu
```

<details><summary>Archives.json output</summary>


</details>

**Run the same image, this time having the image in cache**

<details><summary>Archives.json output</summary>

</details>

**Remove the image**

<details><summary>Archives.json output</summary>

</details>

**Execute an application in a container**
Run `docker run -td ubuntu bash`

<details><summary>Archives.json output</summary>

</details>

**Kill the container previously launched**
Run `docker kill $(docker ps -q)`.

<details><summary>Archives.json output</summary>

</details>
